en.html

```diff
@@ -0,0 +1,472 @@
+
+	<!-- Owl Carousel -->
+	<link type="text/css" rel="stylesheet" href="css/owl.carousel.css" />
+	<link type="text/css" rel="stylesheet" href="css/owl.theme.default.css" />
+
+	<!-- Magnific Popup -->
+	<link type="text/css" rel="stylesheet" href="css/magnific-popup.css" />
+
+	<!-- Font Awesome Icon -->
+	<link rel="stylesheet" href="css/font-awesome.min.css">
+
+	<!-- Custom stlylesheet -->
+	<link type="text/css" rel="stylesheet" href="css/style.css" />
+	
+	<!-- Bootstrap language icons -->
+	<link type="text/css" rel="stylesheet" href="css/languages.min.css" />
+
+	<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+	<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+	<!--[if lt IE 9]>
+		<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+		<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+	<![endif]-->
+</head>
+
+<body>
+	<header id="home">
+		<div class="bg-img" style="background-image: url('./img/mickoflus-title.jpg');">
+			<div class="overlay"></div>
+		</div>
+		<nav id="nav" class="navbar nav-transparent">
+			<div class="container">
+
+				<div class="navbar-header">
+					<div class="navbar-brand">
+						<a href="index.html">
+							<img class="logo" src="img/logo.svg" alt="logo">
+							<img class="logo-alt" src="img/logo-alt.svg" alt="logo">
+						</a>
+					</div>
+
+					<div class="nav-collapse">
+						<span></span>
+					</div>
+				</div>
+
+				<ul class="main-nav nav navbar-nav navbar-right">
+					<li><a href="#home">Introduction</a></li>
+					<li><a href="#about">About Míčkoflusu</a></li>
+					<li><a href="#partners">Partners</a></li>
+					<li><a href="#gallery">Gallery</a></li>
+					<li><a href="#guide">Production guide</a></li>
+					<li><a href="#team">Who's behind it</a></li>
+					<li><a href="#contact">Contact</a></li>
+					<li><a href="index.html"> <span class="lang-sm lang-lbl" lang="cs"></span></a></li>
+				</ul>
+			</div>
+		</nav>
+
+		<div class="home-wrapper">
+			<div class="container">
+				<div class="row">
+
+					<div class="col-md-10 col-md-offset-1">
+						<div class="home-content">
+							<h1 class="white-text text-center">Míčkoflus</h1>
+							<p class="white-text text-center">Shoot them all!</p>
+							<a href="#about"><button class="white-btn">Get more information</button></a>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</header>
+
+	<div id="about" class="section md-padding">
+		<div class="container">
+			<div class="row">
+				<div class="section-header">
+					<h2 class="title">About Míčkoflusu</h2>
+					<p>It drives, shoots it and looks innocent. This is Míčkoflus &#8212;
+					robotics kit from Robotárna (department of <a target="_tab" href="https://helceletka.cz">
+					Youth Centre Helceletova Brno</a>). This kit was firstly introduce on <a href="">RoboticCamp 2018</a>.</p>
```

Comment:
Pokud Helceletka nemá ofiko EN název, nepřekládal bych.

Reply:
Proč by jsi to nepřekládal? Takhle nikdo nebude vědět, co vlastně Helceletka je (jelikož stránky jsou taky jen v češtině).

A když to vezmeš do důsledku, tak `DDM` bys asi v angličtině napsal jako `Youth Centre` ne?

Reply:
Protože to není ofiko název - a co třeba DDM Helceletovat (local youth center) - a nevím, jestli youth je správně (zeptám se)

Reply:
Nějaké novinky? :-)

Reply:
Ping @yaqwsx

Reply:
Mé zdroje lepší překlad neznají, ale intuitivně se jim také nezdá dokonalý. Takže ho použijme.

Reply:
Použíjme tedy v tomto znění `DDM Helceletovat (local youth center)`?

Reply:
Jsem pro ni.